### PR TITLE
Add Google Pixel C

### PR DESCRIPTION
**Pull request by gatoravi**

### CODE REVIEW

File: screens.json [hunk 1]
```diff
@@ -332,6 +332,12 @@
 		"dppx": 2
 	},
 	{
+		"name": "Google Pixel C",
+		"w": 2560,
+		"h": 1800,
+		"ppi": 308,
```
Comment (LeaVerou) [inline screens.json:338]: No diagonal? Also, are you sure ppi is needed? It's automatically calculated, we only include it when it's different than the automatic one.

**Comment by LeaVerou** [comment]
Thanks, see comments!

File: screens.json (hunk 1 above)
Comment (LeaVerou) [inline screens.json:338]: Also this is missing the diagonal...

**Comment by gatoravi** [comment]
Replace this with #146

**Pull request closed without merging**